magefiles/utils.go

```diff
@@ -230,3 +231,60 @@ func cleanupQuay(quayService quay.QuayService, quayOrg string) error {
 	}
 	return nil
 }
+
+func cleanupQuayTags(quayService quay.QuayService, organization, repository string) error {
+	workerCount := 10
+	var wg sync.WaitGroup
+
+	var allTags []quay.Tag
+	errChan := make(chan error)
+
+	page := 1
+	for {
+		tags, hasAdditional, err := quayService.GetTagsFromPage(organization, repository, page)
+		if err != nil {
+			errChan <- fmt.Errorf("error getting tags of `%s` repository of `%s` organization on page `%d`, error: %s", repository, organization, page, err)
```

Comment:
wouldn't this program get stuck here if we really got an error?

Reply:
Yes, I believe you're correct. 
Moving `page++` right after `GetTagsFromPage` should fix the issue, right?

Reply:
I'm not sure whether you can use a channel here, since it's not running in a separate goroutine?

Reply:
You are correct, thanks